cmd/sdkstats/main.go

```diff
@@ -7,46 +7,188 @@ import (
 	"fmt"
 	"io/ioutil"
 	"log"
+	"os"
+	"regexp"
+	"strings"
 
-	"github.com/go-github-metrics-1/pkg/sdkstats"
+	"github.com/learnoperators/go-github-metrics/pkg/sdkstats"
 
 	"github.com/google/go-github/github"
 	"golang.org/x/oauth2"
 )
 
+type helmVersionParser struct{}
+type baseVersionParser struct {
+	searchQ      string
+	searchLatest string
+}
+type gomodVersionParser struct {
+	searchQ string
```

Comment:
Also, would be great have in the README an overview. As for example, describing the purpose of it. After I checked the code shows that we are looking for getting metrics over projects published in GitHub which were done using Operator-SDK. Am I right? 

The starts, commits and etc are nor over operator-sdk repo/project. Am I right?